src/finding/summary.go

```diff
@@ -23,12 +26,18 @@ type Detail struct {
 
 	PackageName    string
 	PackageVersion string
-	CVSS2Score     string
-	CVSS2Vector    string
+	CVSS2          CVSSScore
+	CVSS3          CVSSScore
```

Comment:
Why are two fields required? They seem to be mutually exclusive and the same type, so why not use just one field?

Reply:
They're not mutually exclusive: both may be provided.